gauntlet/packages/gauntlet-solana-contracts/src/lib/constants.ts

```diff
@@ -4,3 +4,5 @@ export const MAX_TRANSACTION_BYTES = 996
 export const ORACLES_MAX_LENGTH = 19
 export const UPGRADEABLE_BPF_LOADER_PROGRAM_ID = new PublicKey('BPFLoaderUpgradeab1e11111111111111111111111')
 export const ADDITIONAL_STATE_BUFFER = 1024
+
+export const TOKEN_DECIMALS = 9
```

Comment:
Would appreciate a quick inline comment here about why we choose 9 decimals as a default, as discussed offline.

Reply:
Left a small comment